applications_dev/search-engine.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  name: search-engine
+  namespace: argocd
+  labels:
+    purpose: trust-anchor
```

Comment:
I guess that should be changed

Reply:
I updated it in marketplace. Is it correct?

Reply:
The labels are mostly used to make filtering easier. Marketplace can be an option or we could introduce a label "search-engine" if it consists of multiple applications.